doc/connectivity/bluetooth/api/hci.txt

```diff
@@ -88,6 +88,8 @@ the controller. By default no vendor events are reported.
 	|       | 1    | Fatal Error                                |
 	|       | 2    | Trace Information Event                    |
 	|       | 3    | Scan Request Received Event                |
+	|       | 4    | Connectionless IQ Report Event             |
```

Comment:
could you increment the rest of the bits?

Reply:
I do not get, which bits should I change?

Reply:
the next ones in the table still have 4, 5 in rfu